config/laravel-logging-channels.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Thoughts: move this back to `laravel-logging.php`, nest them under `channels` key, change the config merge target to `logging`?

I am concerned that we're removing the ability to customise the configuration of these extra logging channels with the current approach

Reply:
Can do but I wasn't sure if it would be confusing changing the schema from laravel default vs. adding out own 🤷‍♂️

Although, the channels are still configured the same way so I think the customisation might already be lost.

I personally don't mind either way so let me know which you'd prefer